release-notes.md

```diff
@@ -318,6 +327,75 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v38.0
+
+This release has been tracked in our [roadmap
+project](https://github.com/orgs/cloud-hypervisor/projects/6) as iteration
+v38.0. The following user visible changes have been made:
+
+### Group Rate Limiter on Block Devices
+
+Users now can throttle a group of block devices with the new
+`--rate-limiter-group` option. Details can be found from the [I/O
+Throttling documentation](docs/io_throttling.md)
+
+### Thread Pinning Support for Block Device Virt-queue
```

Comment:
I think that removing the virt-queue terminology would be better here and just talk about CPU pinning the block device worker thread.